### ISSUE TITLE
Run the workflow in parallel

### ISSUE THREAD

**Issue by DanNixon** [mantidproject/scripts#5]
In [`workflow.py`](https://github.com/mantidproject/scripts/blob/b99b7e584c74f79f5e3f9d44a3233e2155ac80da/development/inelastic/vesuvio_user/vesuvio/workflow.py#L59) there is the loop that iterates over all detectors/spectra/banks which could easily be done in parallel over all cores (or a maximum number of cores given as a flag).

This should wait until I have fixed the workflow tests.

**Comment by ElliotAOram** [comment]
Vesuvio has been moved from the script repository to the main Mantid repo. A [new issue](https://github.com/mantidproject/mantid/issues/16864)  for this has been set up there.
